internal/proto/controller/servers/services/v1/session_service.proto

```diff
@@ -49,14 +49,14 @@ message LookupSessionResponse {
   string tofu_token = 20; // @gotags: `class:"secret"`
   uint32 version = 30; // @gotags: `class:"public"`
   string endpoint = 40; // @gotags: `class:"public"`
-  google.protobuf.Timestamp expiration = 50; // @gotags: `class:"public"`
-  controller.servers.services.v1.SESSIONSTATUS status = 60; // @gotags: `class:"public"`
+  google.protobuf.Timestamp expiration = 50; // @gotags: `class:"public" eventstream:"observation"`
+  controller.servers.services.v1.SESSIONSTATUS status = 60; // @gotags: `class:"public" eventstream:"observation"`
```

Comment:
This status is not recorded in `SessionJobInfo` and `MonitorSessionJobInfo`, but it is here and elsewhere. Should it be recorded everywhere?

Reply:
I think these two protos is mostly for Job proto message and they are not part of the telemetry and I focus more on `ServerWorkerStatus` which will be used as a telemetry. I added the ids for those messages just in case but it wont hurt to have it since it is not customer input